Library/Homebrew/extend/ENV/super.rb

```diff
@@ -313,16 +314,19 @@ def determine_cccfg
   # When passed a block, MAKEFLAGS is removed only for the duration of the block and is restored after its completion.
   sig { params(block: T.nilable(T.proc.returns(T.untyped))).returns(T.untyped) }
   def deparallelize(&block)
-    old = delete("MAKEFLAGS")
+    old_makeflags = delete("MAKEFLAGS")
+    old_make_jobs = delete("HOMEBREW_MAKE_JOBS")
+    self["HOMEBREW_MAKE_JOBS"] = "1"
     if block
       begin
         yield
       ensure
-        self["MAKEFLAGS"] = old
+        self["MAKEFLAGS"] = old_makeflags
+        self["HOMEBREW_MAKE_JOBS"] = old_make_jobs
       end
     end
 
-    old
+    [old_makeflags, old_make_jobs]
```

Comment:
What uses this return value and is this a breaking change?

Reply:
> What uses this return value

Nobody, as far as I can tell. Definitely nothing in Homebrew/core.

> is this a breaking change?

In theory, yes. In practice, given the above, not really. We should maybe just not return anything here.

Reply:
Either retaining the original return value or removing it makes sense to me. Adding a second return value doesn't make much sense.

Reply:
Cool, let's retain the old value:

```suggestion
    old_makeflags
```